examples/README.md

```diff
@@ -0,0 +1,26 @@
+# Sonic Disc Reader
```

Comment:
I don't find a central `README.md` as sustainable since this text file will be growing the more examples are added therefore this will hinder each example's documentation less discoverable. My suggestion is to follow the previous scheme by having a `README.md` in each example folder.

Reply:
This makes sense. I've replaced the original readme files and moved mine into the processing3 sketch folder.